# Bioinformatics

* [tcga-code-tables](https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables)
* [HPO](http://human-phenotype-ontology.github.io/) - Human Phenotype Ontology 人类表型数据库
* [NGS-Workflows](https://jmazz.me/blog/NGS-Workflows)
* [gkno](http://gkno.me) - A bioinformatics pipeline launcher
* [Genenteck](https://www.gene.com/stories/)
* [oncotree](http://oncotree.mskcc.org/#/home) - 肿瘤分类
* [Cancer today](http://gco.iarc.fr/today/home) - 患病率查询
* [GeneMANIA](https://genemania.org/) - GeneMANIA helps you predict the function of your favourite genes and gene set
* [bioDBnet](https://biodbnet-abcc.ncifcrf.gov/) - 基因ID转换
* [inkscape](https://inkscape.org/release/inkscape-1.2/) - 开源的矢量图软件

### pipeline
* [Nextflow](https://www.nextflow.io/docs/latest/index.html)
* [nextflow-demos](https://github.com/stevekm/nextflow-demos) - Demonstrations of various programming techniques for use inside Nextflow pipelines. 
* [awesome-nextflow](https://github.com/nextflow-io/awesome-nextflow)
* [Snakemake](https://snakemake.readthedocs.io/en/stable/index.html)
* [WDL](https://support.terra.bio/hc/en-us/categories/360001399872)
* [CWL](https://www.commonwl.org/)
* [bpipe](http://docs.bpipe.org/)
* [IMPACT-Pipeline](https://impact-pipeline.readthedocs.io/en/latest/)
* [maftools](https://bioconductor.org/packages/release/bioc/vignettes/maftools/inst/doc/maftools.html) - Summarize, Analyze and Visualize MAF Files
* [cfDNApipe](https://github.com/Honchkrow/cfDNApipe) - cfDNApipe(cell free DNA Pipeline) is an integrated pipeline for analyzing cell-free DNA WGBS/WGS data.
* [gonomics](https://pkg.go.dev/github.com/vertgenlab/gonomics) - A collection of genomics software tools written in Go (golang).

### Purity/ploidy
* [sequenza](https://cran.r-project.org/web/packages/sequenza/vignettes/sequenza.html#about) - estimate cancer cellularity, ploidy, genome wide copy number profile and infer for mutated alleles.
* [ABSOLUTE](http://archive.broadinstitute.org/cancer/cga/absolute)  - hbsun
* [PureCN](http://bioconductor.org/packages/release/bioc/html/PureCN.html) - opy number calling and SNV classification using targeted short read sequencing
* [PURPLE](https://github.com/hartwigmedical/hmftools/blob/master/purity-ploidy-estimator/README.md) - 纯度和倍性计算
* [facets](https://github.com/mskcc/facets) - MSKCC的计算纯度和倍性的方法

### Immuno infiltration
* [MCPcounter](https://github.com/ebecht/MCPcounter) - Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression
* [xCell](http://xcell.ucsf.edu/) - cell type enrichment analysis from gene expression data for 64 immune and stroma cell types
* [GSVA](https://github.com/rcastelo/GSVA) - Gene set variation analysis for microarray and RNA-seq data
* [CIBERSORT](https://cibersort.stanford.edu/) - an estimation of the abundances of member cell types in a mixed cell population, using gene expression data.
* [TIDE](https://github.com/liulab-dfci/TIDEpy) - 用RNAseq数据计算免疫逃逸的能力
* [TIMER](http://timer.cistrome.org/) - 免疫浸润计算
* [IOBR](https://github.com/IOBR/IOBR) - 集合多种计算免疫微环境的包
* [ESTIMATE](http://bioinformatics.mdanderson.org/main/ESTIMATE:Overview) - is a tool for predicting tumor purity, and the presence of infiltrating stromal/immune cells in tumor tissues using gene expression data. 

### Genome-related web
* [CDC](https://www.cdc.gov/)
* [transvarweb](http://bioinformatics.mdanderson.org/transvarweb/) MD-anderson开发的注释工具
* [Gencode](http://www.gencodegenes.org/)
* [Ensemble](http://www.ensembl.org/index.html)
* [Uniprob](http://www.uniprot.org/)
* [Genecards](http://www.genecards.org/)
* [EMBL-EBI](https://www.ebi.ac.uk/)
* [UCSC](http://genome.ucsc.edu/)
* [Genome Reference Consortium](https://www.ncbi.nlm.nih.gov/grc)
* [HGNC](http://www.genenames.org/) - 人类基因组命名委员会
* [ENCODE](https://www.encodeproject.org/)
* [cbioportal](http://www.cbioportal.org/) -  **visualization, analysis and download of large-scale cancer genomics data sets**
* [awesome-bio-datasets](https://github.com/OpenGene/awesome-bio-datasets)
* [pharmgkb](https://www.pharmgkb.org/) - 药物基因组
* [DrugBank](https://www.drugbank.ca/) - 药物数据库
* [DGV](http://dgv.tcag.ca/dgv/app/home) - 收录了健康人群中序列长度大于50bp的结构变异。

### Database
* [ExAC](http://exac.broadinstitute.org/)
* [ESP6500](http://evs.gs.washington.edu/EVS/)
* [1000 Genome](http://www.internationalgenome.org/home)
* [GenomeAD](http://gnomad.broadinstitute.org/) - 最全人群频率数据库
* [awesome-bio-datasets](https://github.com/OpenGene/awesome-bio-datasets)
* [cngb](https://db.cngb.org/pird/home/) - 华大数据库
* [Orphanet](http://www.orpha.net/consor/cgi-bin/index.php?lng=EN) - 罕见病数据库
* [jaspar](https://jaspar2022.genereg.net/docs/) - JASPAR is a regularly maintained open-access database storing manually curated TF binding preferences as position frequency matrices (PFMs).

### Cancer database
* [CIVIC](https://civic.genome.wustl.edu/home)
* [COSMIC](http://cancer.sanger.ac.uk/cosmic)
* [OncoKB](http://oncokb.org)
* [awesome-cancer-variant-databases](https://github.com/seandavi/awesome-cancer-variant-databases)
* [PMKB](https://pmkb.weill.cornell.edu/) - 突变基因临床查询
* [varsome](https://varsome.com/) - 突变查询数据库

### Protein
* [Uniprot](http://www.uniprot.org/)
* [Pfam](http://pfam.xfam.org/)
* [PBD](http://www.rcsb.org/pdb/home/home.do)
* [HPA](http://www.proteinatlas.org/) - Human Protein Atlas 数据库
* [Amino acid distance](https://en.wikipedia.org/wiki/Amino_acid_replacement) - 氨基酸之间的距离计算
* [OmicsEV](https://github.com/bzhanglab/OmicsEV) - A tool for large scale omics datasets evaluation

### Annotation
* [annovar](http://annovar.openbioinformatics.org/en/latest/)
* [Oncotator](http://portals.broadinstitute.org/oncotator/)
* [HGVS](http://varnomen.hgvs.org/) - 基因组注释说明
* [snpeff](http://snpeff.sourceforge.net/index.html) - Genomic variant annotations and functional effect prediction toolbox. 
* [Genomic variant annotations and functional effect prediction toolbox. ](https://genomemedicine.biomedcentral.com/articles/10.1186/gm543)
* [trackViewer](https://bioconductor.org/packages/release/bioc/html/trackViewer.html)
* [mygene](http://docs.mygene.info/en/latest/index.html) - 基因ID转换
* [transvar](https://transvar.readthedocs.io/en/latest/index.html) - MD Anderson开发的注释工具
* [vep](https://grch37.ensembl.org/info/docs/tools/vep/script/index.html)


### QC
* [trimmomatic](http://www.usadellab.org/cms/index.php?page=trimmomatic) - 去接头软件
* [trim_galore ](https://github.com/FelixKrueger/TrimGalore) - 质控去接头软件，包装了cutadapt和fastqc，可以自动识别接头
* [cutadapt](https://github.com/marcelm/cutadapt) - 去接头软件 
* [fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) - 质控软件
* [fastp](https://github.com/OpenGene/fastp) - 质控软件
* [AfterQC](https://github.com/OpenGene/AfterQC) - 质控软件
* [MultiQC](https://github.com/ewels/MultiQC) - 整合QC质控结果
* [Conpair](https://github.com/nygenome/Conpair) - 计算肿瘤样本VS对照样本配对和污染情况
* [contatester](https://github.com/CNRGH/contatester) - Compute the Allelic Balance of a sample from a VCF file, check if a cross human contamination is present and estimate the degree of contamination.
* [preseq](https://github.com/smithlabcode/preseq) - Software for predicting library complexity and genome coverage in high-throughput sequencing.

### UMI
* [ConsensusCruncher](https://github.com/pughlab/ConsensusCruncher) - ConsensusCruncher is a tool that suppresses errors in next-generation sequencing data by using unique molecular identifers (UMIs) to amalgamate reads derived from the same DNA template into a consensus sequence.
* [fgbio](http://fulcrumgenomics.github.io/fgbio/tools/latest/)
* [UMI-tools](https://github.com/CGATOxford/UMI-tools)

### SNV
* [MuSE](http://bioinformatics.mdanderson.org/main/MuSE)  MD Anderson
* [SomaticSniper](http://gmt.genome.wustl.edu/packages/somatic-sniper/)
* [strelka](https://github.com/Illumina/strelka)
* [MutScan](https://github.com/OpenGene/MutScan) - Detect and visualize target mutations by scanning FastQ files directly
* [VarScan](http://varscan.sourceforge.net/)
* [VarDict](https://github.com/AstraZeneca-NGS/VarDict)
* [samtools](http://www.htslib.org/doc/#manual-pages)
* [freebayes](https://github.com/ekg/freebayes) - a haplotype-based variant detector
* [Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/4409917447707-Mutect2)

### CNV

* [ascat](https://github.com/Crick-CancerGenomics/ascat) - Allele-specific copy number analysis of tumors
* [GATK](https://software.broadinstitute.org/gatk/)
* [bedtools](http://bedtools.readthedocs.io/en/latest/) -  a powerful toolset for genome arithmetic
* [HMMcopy](http://www.bioconductor.org/packages/release/bioc/html/HMMcopy.html) 
* [facets](https://sites.google.com/site/mskfacets/) - CNV for WES
* [cnv_facets](https://github.com/dariober/cnv_facets) - 包装的FACETS软件，用于计算CNV、纯度、倍性
* [CNVkit](http://cnvkit.readthedocs.io/en/stable/pipeline.html) - CNV分析，基于on-target和off-target
* [CopywriteR](http://www.bioconductor.org/packages/release/bioc/html/CopywriteR.html) - CNV分析，基于off-target
* [DNAcopy](http://www.bioconductor.org/packages/release/bioc/html/DNAcopy.html) - CNV分析
* [CONTRA](http://contra-cnv.sourceforge.net/) - 目标区域CNV分析
* [copyCat](https://github.com/chrisamiller/copycat) - somatic copy number aberrations
* [ExomeCNV](https://secure.genome.ucla.edu/index.php/ExomeCNV_User_Guide) - CNV (Copy-Number Variants) and LOH (Loss of Heterozygosity) from exome sequencing data
* [ichorCNA](https://github.com/broadinstitute/ichorCNA) - estimating the fraction of tumor in cell-free DNA from ultra-low-pass whole genome sequencing (ULP-WGS, 0.1x coverage)
* [PureCN](http://bioconductor.org/packages/release/bioc/html/PureCN.html) - This package estimates tumor purity, copy number, and loss of heterozygosity (LOH), and classifies single nucleotide variants (SNVs) by somatic status and clonality.
* [sequenza](https://cran.r-project.org/web/packages/sequenza/vignettes/sequenza.html) - Copy Number Estimation from Tumor Genome Sequencing Data
* [ascets](https://github.com/beroukhim-lab/ascets) - Arm-level Somatic Copy-number Events in Targeted Sequencing
* [CNTools](https://www.bioconductor.org/packages/release/bioc/html/CNTools.html) - Convert segment data into a region by sample matrix to allow for other high level computational analyses.

### SV
* [pindel](https://github.com/genome/pindel) - detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications 
* [breakdancer](http://breakdancer.sourceforge.net/) - genome-wide detection of structural variants from next generation paired-end sequencing reads
* [lumpy](https://github.com/arq5x/lumpy-sv) - A probabilistic framework for structural variant discovery
* [manta](https://github.com/Illumina/manta) - Structural variant and indel caller
* [SVision](https://github.com/xjtu-omics/SVision) - Detecting genome structural variants with deep learning in single molecule sequencing


### Gene fusion
* [GeneFuse](https://github.com/OpenGene/GeneFuse) - Gene fusion detection and visualization
* [svict](https://github.com/vpc-ccg/svict) - Structural Variation and fusion detection using targeted sequencing data from circulating cell free DNA
* [FusionMap](http://www.arrayserver.com/wiki/index.php?title=FusionMap)
* [FACTERA](https://factera.stanford.edu/)
* [STAR-Fusion](https://github.com/STAR-Fusion/STAR-Fusion/wiki)
* [arriba](https://github.com/suhrig/arriba) - Fast and accurate gene fusion detection from RNA-Seq data
* [FusionCatcher](https://github.com/ndaniel/fusioncatcher) - Finder of Somatic Fusion Genes in RNA-seq data
* [ShatterSeek](https://github.com/parklab/ShatterSeek) - 评估染色体碎裂(chromothripsis)
* [ChainFinder]() - chromoplexy（一种复杂重排）
* [EasyFuse](https://github.com/TRON-Bioinformatics/EasyFuse) - EasyFuse is a pipeline for accurate fusion gene detection from RNA-seq data. 2022
* [MapSplice2](https://github.com/davidroberson/MapSplice2) (2.2.1)
* [InFusion](https://bitbucket.org/kokonech/infusion/src/master/) (0.8)
* [SOAPfuse](https://sourceforge.net/projects/soapfuse/) (1.2.7) 

### MSI
* [Msisensor](https://github.com/ding-lab/msisensor)
* [MANTIS](https://github.com/OSU-SRLab/MANTIS) - Microsatellite Analysis for Normal-Tumor InStability
* [VisualMSI](https://github.com/OpenGene/VisualMSI)

### HRD
* [scarHRD](https://github.com/sztup/scarHRD) - scarHRD is an R package which determines the levels of homologous recombination deficiency (telomeric allelic imbalance, loss off heterozygosity, number of large-scale transitions) based on NGS (WES, WGS) data.

### RNA
* [rna-seq-strand](https://github.com/igordot/genomics/blob/master/notes/rna-seq-strand.md) - RNA分析链向选择
* [Hisat2](http://ccb.jhu.edu/software/hisat2/index.shtml) - graph-based alignment of next generation sequencing reads to a population of genomes
* [stringtie](http://www.ccb.jhu.edu/software/stringtie/) - Transcript assembly and quantification for RNA-Seq
* [gffread](http://ccb.jhu.edu/software/stringtie/gff.shtml#gffread) - 通过gff文件提取转录本fasta序列
* [Xfam](http://rfam.xfam.org/) - RNA数据库
* [featureCounts](http://bioinf.wehi.edu.au/featureCounts/) - map到基因的reads统计
* [htseq](http://htseq.readthedocs.io/en/release_0.9.1/) - map到基因的reads统计htseq-count
* [miRBase](http://www.mirbase.org/) - miRNA数据库
* [STAR](https://github.com/alexdobin/STAR) - RNAseq mapping tools
* [STAR-Fusion](https://github.com/STAR-Fusion/STAR-Fusion/wiki) - 基因融合分析工具
* [arriba](https://github.com/suhrig/arriba) - Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.
* [Trinity_CTAT](https://github.com/NCIP/Trinity_CTAT/wiki) - Trinity Cancer Transcriptome Analysis Toolkit 
* [Trinity](https://github.com/trinityrnaseq/trinityrnaseq/wiki) - RNA-Seq De novo Assembly
* [Cufflinks](http://cole-trapnell-lab.github.io/cufflinks/) - Transcriptome assembly and differential expression analysis for RNA-Seq
* [GimmeMotifs](http://gimmemotifs.readthedocs.io/en/master/) - motif analysis
* [gkmsvm](http://www.beerlab.org/gkmsvm/) - Enhanced Regulatory Sequence Prediction Using Gapped k-mer Features
* [meme](http://meme-suite.org/index.html) - Motif-based sequence analysis tools
* [rnacocktail](https://bioinform.github.io/rnacocktail/) - RNA分析流程 DOI: 10.1038/s41467-017-00050-4
* [MSigDB](http://software.broadinstitute.org/gsea/msigdb) -  a collection of annotated gene sets for use with GSEA software
* [rbsurv](http://www.bioconductor.org/packages/release/bioc/html/rbsurv.html) - This package selects genes associated with survival. 获取与生存时间相关的基因
* [WGCNA](https://cran.r-project.org/web/packages/WGCNA/index.html) - WGCNA其译为加权基因共表达网络分析。该分析方法旨在寻找协同表达的基因模块(module)，并探索基因module与关注的表型之间的关联关系，以及网络中的核心基因。http://www.stat.wisc.edu/~yandell/statgen/ucla/WGCNA/wgcna.html
* [CIBERSORT ](https://cibersort.stanford.edu/) - to provide an estimation of the abundances of member cell types in a mixed cell population, using gene expression data.
* [RSEM](https://github.com/deweylab/RSEM) - RNA分析工具

### Genome assembly
* [platanus](http://platanus.bio.titech.ac.jp)  复杂基因组组装
* [allpath](http://software.broadinstitute.org/allpaths-lg/blog/)  
* [MECAT](https://github.com/xiaochuanle/MECAT) - 三代组装软件
* [megahit](https://github.com/voutcn/megahit) - meta组装，速度快内存消耗少
* [wtdbg](https://github.com/lh3/wtdbg2) - 基因组组装 by 李恒

### Mapping
* [BWA](http://bio-bwa.sourceforge.net/) - BWA is a software package for mapping low-divergent sequences against a large reference genome
* [BBmap](https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbmap-guide/)
* [GSNAP](http://research-pub.gene.com/gmap/) - Genomic Short-read Nucleotide Alignment Program
* [DRAGMAP](https://github.com/Illumina/DRAGMAP) - Illumina Dragmap is the Dragen mapper/aligner Open Source Software.
* [alignment_tools](https://en.m.wikipedia.org/wiki/List_of_sequence_alignment_software) - alignment工具总结

### Predict genes
* [AUGUSTUS](http://bioinf.uni-greifswald.de/augustus/) - AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences
* [TransDecoder](https://github.com/TransDecoder/TransDecoder) - CDS预测

### TCR

* [MIGEC](http://migec.readthedocs.io/en/latest/index.html) - Molecular Identifier Guided Error Correction pipeline
* [igblast](https://www.ncbi.nlm.nih.gov/igblast/) - NCBI开发的用于比对TCR的工具
* [vdjtools](https://vdjtools-doc.readthedocs.io/en/master/index.html) - TCR分析套件
* [vdjmatch](https://github.com/antigenomics/vdjmatch) - Matching T-cell repertoire against a database of TCR antigen specificities
* [gliph](https://github.com/immunoengineer/gliph) - GLIPH clusters TCRs that are predicted to bind the same MHC-restricted peptide antigen.
* [iSMART](https://github.com/s175573/iSMART) - immuno-Similarity Measurement by Aligning Receptors of T cells
* [TRUST4](https://github.com/liulab-dfci/TRUST4) - 使用RNAseq数据分析TCR和BCR
* [VDJdb](https://vdjdb.cdr3.net/search) - CDR3-抗原数据库
* [immunarch](https://immunarch.com/) - Fast and Seamless Exploration of Single-cell and Bulk T-cell/Antibody Immune Repertoires in R
* [TRUST4](https://github.com/liulab-dfci/TRUST4) - TRUST is a computational tool to analyze TCR and BCR sequences using unselected RNA sequencing data


### Evolution
* [phangorn](https://github.com/KlausVigo/phangorn) - 肿瘤进化算法
* [CITUP](https://github.com/sfu-compbio/citup) - Clonality Inference in Multiple Tumor Samples using Phylogeny
* [deconstructSigs](https://github.com/raerose01/deconstructSigs) - R语言 signature分析
* [LICHeE](http://viq854.github.io/lichee/) - Fast and scalable inference of multi-sample cancer lineages
* [ExpressBetaDiversity](https://github.com/dparks1134/ExpressBetaDiversity) - 可基于tree计算beta多样性
* [phangorn](https://cran.r-project.org/web/packages/phangorn/index.html) - 肿瘤进化
* [clonevol](https://github.com/hdng/clonevol) - Inferring and visualizing clonal evolution in multi-sample cancer sequencing
* [dndscv](https://github.com/im3sanger/dndscv) - 计算肿瘤样本dn/ds
* [btctools](https://github.com/josephhughes/btctools) - 计算ds/dn
* [treeomics](https://github.com/reiterlab/treeomics) - 构建进化树
* [MACHINA](https://github.com/raphael-group/machina) - 构建进化树
* [MOBSTER](https://caravagnalab.github.io/mobster/index.html) - 克隆进化2020
* [sciclone](https://github.com/genome/sciclone) - 克隆进化分析
* [pyclone](http://compbio.bccrc.ca/software/pyclone/) - 克隆进化分析
* [citup](https://github.com/amcpherson/citup) - The following package implements the method described in Clonality inference in multiple tumor samples using phylogeny
* [TimeScape](https://bioconductor.org/packages/release/bioc/vignettes/timescape/inst/doc/timescape_vignette.html) - TimeScape is a visualization tool for temporal clonal evolution.
* [SCHISM](https://github.com/KarchinLab/SCHISM) - SCHISM is a computational tool designed to infer subclonal hierarchy and the tumor evolution from somatic mutations.
* [EstimateClonality](https://bitbucket.org/nmcgranahan/pancancerclonality/src/master/EstimateClonality/)
* [ECLIPSE](https://github.com/amf71/ECLIPSE) - R package for clonal deconvolution of tumour-informed ctDNA data using clonality and copy number information from tumour tissue. 

### Neoantigen
* [MuPeXI](https://github.com/ambj/MuPeXI) - Mutant Peptide eXtractor and Informer
* [LOHHLA](https://bitbucket.org/mcgranahanlab/lohhla/src/master/)

### Tools

* [STRING](https://string-db.org/cgi/input.pl?UserId=1hcUYxDKX0aK&sessionId=xykqPVdtcqrE&input_page_show_search=off) - 蛋白互作
* [SetRank](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1571-6) - 富集分析
* [GSVA](http://bioconductor.org/packages/release/bioc/html/GSVA.html) - 富集分析，可以进行ssGSEA分析
* [bam-readcount](https://github.com/genome/bam-readcount) - 位点深度统计
* [somaticfreq](https://github.com/PoisonAlien/somaticfreq) - knowledge-based genotyping of targetted somatic variants from the tumor BAM file
* [Wgsim](https://github.com/lh3/wgsim) - 数据模拟
* [bamgineer](https://github.com/pughlab/bamgineer) - 数据模拟
* [vcf2bed](http://bedops.readthedocs.io/en/latest/content/reference/file-management/conversion/vcf2bed.html)
* [slicer](https://github.com/OpenGene/slicer) - Slice a text file (like FastQ) to smaller files by lines, with gzip supported
* [Vt](https://genome.sph.umich.edu/wiki/Vt) - a variant tool set that discovers short variants from Next Generation Sequencing data.
* [hgvs](https://github.com/biocommons/hgvs) - 解析hgvs
* [deeptools](https://deeptools.readthedocs.io/en/develop/index.html) - tools for exploring deep sequencing data
* [sambamba](https://github.com/biod/sambamba) - bam文件处理工具，类似samtools，速度快
* [BamUtil](https://genome.sph.umich.edu/wiki/BamUtil) - bamUtil is a repository that contains several programs that perform operations on SAM/BAM files. 
* [bamtools](https://github.com/pezmaster31/bamtools) - A small, but powerful suite of command-line utility programs for manipulating and querying BAM files.
* [ngs-bits](https://github.com/imgag/ngs-bits) - Short-read sequencing tools (SampleGender)

### Web
* [CCB](https://ccb.jhu.edu/software.shtml) - 约翰霍普金斯大学计算生物学研究中心(RNA分析工具、meta分析工具)
* [quinlanlab](http://quinlanlab.org/#portfolioModal6) - Quinlan实验室，分析工具
* [raphael lab](http://compbio.cs.brown.edu/software) - raphael lab 实验室
* [学习网站](https://www.plob.org/) 
* [cgdsr](https://cran.r-project.org/web/packages/cgdsr/index.html) - TCGA临床数据下载 
* [cbioportal](http://www.cbioportal.org/) - TCGA临床数据下载,数据可视化MutationMapper、OncoPrinter
* [perl计算序列相似性](http://www.perlmonks.org/?node_id=500235) - 计算序列的相似性perl
* [intogen](https://www.intogen.org/search) - 可视化


### Microbe/Virus
* [kraken](http://ccb.jhu.edu/software/kraken/MANUAL.html) - Kraken taxonomic sequence classification system
* [microbiology](http://microbiology.se/) - 一些分析软件
* [Parsnp](https://harvest.readthedocs.io/en/latest/index.html) - 细菌/病毒的call SNP
* [fastv](https://github.com/OpenGene/fastv) - 病毒检测

### Methylation
* [ChAMP](http://www.bioconductor.org/packages/release/bioc/vignettes/ChAMP/inst/doc/ChAMP.html) - [甲基化R包](https://blog.csdn.net/joshua_hit/article/details/54982018)
* [Bismark](https://github.com/FelixKrueger/Bismark) - 甲基化比对工具
* [RESET](http://ciriellolab.org/reset/reset.html) - TCGA Pan-cancer甲基化分析工具
* [MONOD2](https://github.com/dinhdiep/MONOD2) - 甲基化单体型
* [MethPipe](http://smithlabresearch.org/software/methpipe/) - a computational pipeline for analyzing bisulfite sequencing data
* [InfiniumPurify](https://cran.r-project.org/web/packages/InfiniumPurify/index.html) - Estimate and Account for Tumor Purity in Cancer Methylation Data Analysis
* [minfi](https://bioconductor.org/packages/release/bioc/html/minfi.html) - Analyze Illumina Infinium DNA methylation arrays
* [GREAT](http://great.stanford.edu/public/html/) - GREAT predicts functions of cis-regulatory regions.

### Driver gene
* [mutsigcv](http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/mutsigcv) - 驱动基因
* [OncodriveFML](https://bitbucket.org/bbglab/oncodrivefml)
* [CovGen](https://github.com/tgen/CovGen) - Creates a target specific exome_full192.coverage.txt file required by MutSig

### plot
* [motifStack](https://bioconductor.org/packages/release/bioc/html/motifStack.html) motif绘图
* [homer](http://homer.ucsd.edu/homer/) - motif
* 
### miRNA
* [miRbase](http://www.mirbase.org/) - the microRNA database

### single-cell

* [scrna-tools](https://www.scrna-tools.org/
* [awesome-single-cell](https://github.com/seandavi/awesome-single-cell)
* [single-cell-tutorial](https://github.com/theislab/single-cell-tutorial) - Scripts for "Current best-practices in single-cell RNA-seq: a tutorial"
* [tracer](https://github.com/Teichlab/tracer) - 单细胞RNAseq测序的TCR分析
* [awesome-single-cell](https://github.com/seandavi/awesome-single-cell) - List of software packages (and the people developing these methods) for single-cell data analysis
* [SingleCell](https://github.com/SGI-LungCancer/SingleCell) - 计算CNV score，可用于区分malignant cell
* [infercnv](https://github.com/broadinstitute/infercnv) - Inferring copy number alterations from tumor single cell RNA-Seq data
* [infercnvpy](https://github.com/icbi-lab/infercnvpy) - Scanpy plugin to infer copy number variation (CNV) from single-cell transcriptomics data
* [copykat](https://github.com/navinlabcode/copykat/) - Inference of genomic copy number and subclonal structure of human tumors from high-throughput single cell RNAseq data
* [batchbench](https://github.com/cellgeni/batchbench) - BatchBench is a Nextflow workflow for running the following scRNA-Seq data batch effect correction methods
* [dyno](https://github.com/dynverse/dyno) - 60种轨迹分析
* [DoubletFinder](https://github.com/chris-mcginnis-ucsf/DoubletFinder) - 过滤doublets细胞
* [DoubletDecon](https://github.com/EDePasquale/DoubletDecon) - 基于反卷积方法去除doublets细胞
* [DropletUtils](https://bioconductor.org/packages/release/bioc/html/DropletUtils.html) - Utilities for handling droplet-based single-cell RNA-seq data
* [AUCell](https://bioconductor.org/packages/release/bioc/vignettes/AUCell/inst/doc/AUCell.html) - 单细胞的Gene set分析
* [GENIE3](https://bioconductor.org/packages/release/bioc/vignettes/GENIE3/inst/doc/GENIE3.html) - Infer gene regulatory network (GRNs) based on co-expression patterns
* [RcisTarget](https://bioconductor.org/packages/release/bioc/vignettes/RcisTarget/inst/doc/RcisTarget.html) - Transcription factor binding motif enrichment
* [pySCENIC](https://github.com/aertslab/pySCENIC) - pySCENIC is a lightning-fast python implementation of the SCENIC pipeline which enables biologists to infer transcription factors, gene regulatory networks and cell types from single-cell RNA-seq data.
* [scvelo](https://github.com/theislab/scvelo) - RNA velocity generalized through dynamical modeling
* [Velocyto ](https://github.com/velocyto-team/velocyto.py) - Velocyto is a library for the analysis of RNA velocity.
* [monocle3](https://cole-trapnell-lab.github.io/monocle3/) - An analysis toolkit for single-cell RNA-seq.
* [harmony](https://portals.broadinstitute.org/harmony/) - 主要用于数据整合
* [cellphonedb](https://github.com/Teichlab/cellphonedb) - 细胞通讯
* [cellchat](http://www.cellchat.org/) - 细胞通讯
* [stlearn](https://stlearn.readthedocs.io/en/latest/#) - a downstream analysis toolkit for Spatial Transcriptomics data
* [scFusion](https://github.com/XiDsLab/scFusion) - scFusion is a computational pipeline for detecting gene fusions at single-cell resolution. scFusion works on Linux/Mac OS

### signature
* [MutationalPatterns](https://bioconductor.org/packages/release/bioc/html/MutationalPatterns.html) - An extensive toolset for the characterization and visualization of a wide range of mutational patterns in base substitution catalogs.
* [deconstructSigs](https://github.com/raerose01/deconstructSigs) - signature分析
* [SparseSignatures](https://bioconductor.org/packages/release/bioc/html/SparseSignatures.html) - Mutational signature 

### Chip-seq
* [samr](https://cran.r-project.org/web/packages/samr/index.html) - 芯片数据分析，置换检验，没有limma的稳健性好
* [limma](http://www.bioconductor.org/packages/release/bioc/html/limma.html)
* [HOMER](https://github.com/IGBIllinois/HOMER) - HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and ChIP-Seq analysis. 

### Cluster
* [iClusterPlus](https://bioconductor.org/packages/release/bioc/html/iClusterPlus.html) - integrative clustering of multiple genomic data sets
* [CMSclassifier](https://github.com/Sage-Bionetworks/CMSclassifier) - 肠癌分型

### NIPT
* [wisecondor](https://github.com/VUmcCGP/wisecondor) - Detect fetal trisomies and smaller CNV's in a maternal plasma sample using whole-genome data.


